carma/launch/plugins.launch.py

```diff
@@ -451,13 +454,42 @@ def generate_launch_description():
                 ],
                 parameters=[
                     vehicle_characteristics_param_file, #vehicle_response_lag
-                    pure_pursuit_tuning_parameters, #pure_pursuit calibration parameters
                     vehicle_config_param_file
                 ]
             ),
         ]
     )
 
+    carma_trajectory_follower_wrapper_container = ComposableNodeContainer(
+        package='carma_ros2_utils',
+        name='carma_trajectory_follower_wrapper_container',
+        executable='carma_component_container_mt',
+        namespace=GetCurrentNamespace(),
+        composable_node_descriptions=[
+            ComposableNode(
+                    package='trajectory_follower_wrapper',
+                    plugin='trajectory_follower_wrapper::TrajectoryFollowerWrapperNode',
+                    name='trajectory_follower_wrapper',
+                    extra_arguments=[
+                    {'use_intra_process_comms': True},
+                    {'--log-level' : GetLogLevel('trajectory_follower_wrapper', env_log_levels) }
+                ],
+                remappings = [
+                    ("plugin_discovery", [ EnvironmentVariable('CARMA_GUIDE_NS', default_value=''), "/plugin_discovery" ] ),
+                    ("ctrl_raw", [ EnvironmentVariable('CARMA_GUIDE_NS', default_value=''), "/ctrl_raw" ] ),
+                    ("trajectory_follower_wrapper/plan_trajectory", [ EnvironmentVariable('CARMA_GUIDE_NS', default_value=''), "/plugins/pure_pursuit/plan_trajectory" ] ),
+                    ("current_pose", [ EnvironmentVariable('CARMA_LOCZ_NS', default_value=''), "/current_pose" ] ),
+                    ("vehicle/twist", [ EnvironmentVariable('CARMA_INTR_NS', default_value=''), "/vehicle/twist" ] ),
+                ],
+                parameters=[
+                    vehicle_characteristics_param_file, #vehicle_response_lag
+                    pure_pursuit_tuning_parameters, #pure_pursuit calibration parameters
```

Comment:
Are we using the vehicle characteristics and pure_pursuit params?
They are not declared or loaded in the wrapper node